main.py

```diff
@@ -0,0 +1,35 @@
+class Fridge:
+    instance = None
+
+    def __init__(self, brand="Bosch", model="CC11", capacity_in_liters=15, is_defrosing=False,
+                 energy_efficiency_class='A'):
+        self.__brand = brand
+        self.__model = model
+        self.__capacity_in_liters = capacity_in_liters
+        self.__is_defrosing = is_defrosing
+        self.__energy_efficiency_class = energy_efficiency_class
+
+    def turn_on_defrosing(self):
+        self.__is_defrosing = True
+
+    def turn_off_defrosing(self):
+        self.__is_defrosing = False
+
+    def delete_model_info(self):
+        del self.__model
```

Comment:
"del" command removes attribute entirely from memory. I suppose you would want just to make it None or empty string